react/javascript/demo-core-sdk/src/demos/Dashboards.jsx

```diff
@@ -72,21 +74,21 @@ export const Dashboards = () => {
   const columns = [
     {
       id: 'id',
+      size: 'small',
       title: 'ID',
       type: 'string',
```

Comment:
uh oh. is this that key sorting linter rule? Going to need to disable that if it gets reset in sdk-codegen